config.php

```diff
@@ -120,12 +120,16 @@
 
 // DEBUGGING & DEVELOPERS
 $this->showStats = false;
-$this->debug = false;
+$this->debug = true; //false;
```

Comment:
That's a fair point. One way to handle this is to have a release script which sets reasonable defaults for production users.